scraper/prototype.py

```diff
@@ -18,6 +19,30 @@
 logger = init_logger()
 
 def main():
+    if len(sys.argv) == 1:
+        scrape()
+    else:
+        scrape_one(sys.argv[1:])
+
+
+def get_start_date():
+    return dt.datetime.now().isoformat()[:10]
```

Comment:
Ce serait plus propre de faire ça ici :
```suggestion
    return dt.date.today().isoformat()
```

Reply:
Corrigé.
(J'avais simplement couper / coller la ligne de code existante).